metadata-ingestion/src/datahub/ingestion/source/looker/looker_config.py

```diff
@@ -209,12 +209,17 @@ class LookerDashboardSourceConfig(
     )
     extract_independent_looks: bool = Field(
         False,
-        description="Extract looks which are not part of any Dashboard. To enable this flag the stateful_ingestion should also be enabled.",
+        description="Extract looks which are not part of any Dashboard. To enable this flag the stateful_ingestion "
+        "should also be enabled.",
     )
     emit_used_explores_only: bool = Field(
         True,
         description="When enabled, only explores that are used by a Dashboard/Look will be ingested.",
     )
+    include_looker_element_in_platform_instance: bool = Field(
+        True,
+        description="When enabled, platform instance will be added in dashboard and chart urn.",
+    )
```

Comment:
@sid-acryl If this defaults to true then it will change URNs of all existing recipes. Is there a migration that happens automatically in this case?

Reply:
Yup it will change urn of all existing recipes. Not migration happens automatically, let me know if I make it default to False

Reply:
@anshbansal ?